### DISCUSSION TITLE
Stalls on ws4py : INFO Using epoll

### DISCUSSION THREAD

**Discussion by capnjip** [blakeblackshear/frigate#5096] [Ask A Question]
Hello, me again.

I've been fighting with my reolink camera setup and I can't get past this stage in start up. I can let this sit and wait for as long as 10min, but nothing happens.
 
![ws4py stalled](https://user-images.githubusercontent.com/96850819/212504670-2f303a21-42f5-407b-9244-4789578189e1.PNG)

frigate.yml:
```## First install, 13Jan2022. Following https://docs.frigate.video/guides/getting_started "creating a config file"

mqtt:
  host: 192.168.3.10
  port: 1883
  user: <omitted>
  password: <omitted>

detectors:
  coral:
    type: edgetpu
    device: pci:0

cameras:
  sidedoor:
    ffmpeg:
      inputs:
        - path: rtsp://<omitted>:<omitted>@192.168.xx.xx:554//h264Preview_01_sub
#        - path: rtmp://192.168.XX.xx/bcs/channel0_sub.bcs?channel=0&stream=0&user=<omtted>&password=<omitted>
          roles:
#            - record
            - detect
            - rtmp

    rtmp:
      enabled: false
    detect:
#      width: 2034
      width: 640
#      height: 1296
      height: 480
```

**Comment by NickM-27** [comment]
What do you expect to happen? This looks like a standard frigate logs with no errors and running great

**Reply by capnjip** [reply]
Oh... well, I suppose I was waiting for the command prompt to come back. It does work in home assistant. Thank you

**Reply by NickM-27** [reply]
you need to run `docker compose up -d` for that, otherwise the command prompt remains just showing any logs that come from frigate
